docs/specs/git.yml

```diff
@@ -403,6 +403,7 @@ repos:
         a.md:
 outputs:
   .errors.log: |
+    ["error","committish-not-found","Can't find branch, tag, or commit 'live' for repo *"]
```

Comment:
This error was raised from `docfx restore`, previous implementation does not raise this error which should be an bug

Reply:
Is this error duplicated with the one below?

Reply:
Nop, the error from `docfx build` contains file info